lib/conditional.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+var utils = require('./utils.js');
+
+function add_condition(block, condition){
+   /*
+      Adds a condition to a block and its children if relevant.
+
+      @param block: an esprima node
+      @param condition: a string to set as condition property
+   */
+   if (block.type === 'BlockStatement' || block.type === 'ForStatment'){
```

Comment:
You should break these strings out as Constants somewhere, so we don't have random strings floating around.